### DISCUSSION TITLE
Scenario option: Only allow one of each ride type

### DISCUSSION THREAD

**Discussion by deurklink** [OpenRCT2/OpenRCT2#19287] [Feature requests]
An easy way to cheese most scenarios in Rollercoaster Tycoon is to spam the same ride over and over again. This is often done with launched freefall rides or small corkscrew coasters. This works in most scenarios, and is usually a great way to beat them.

One way to counter this cheese tactic would be by making a new scenario option where you only allow each ride type to be built only once. You could also make an option for maximum of two of the same ride type, so you can still make dueling rides. I guess this could go in the park options: 
![image](https://user-images.githubusercontent.com/6469413/215264658-2264d8a4-9cca-404a-a58a-152b90af1165.png)

I think having this feature would be a good way to create scenarios that are more difficult to cheese.

**Comment by ocalhoun6** [comment]
There's another wrinkle to this, allowing scenario designers to make it even more difficult if they want to.

If you start the player off with a limited number of rides invented *and* use this new restriction, you could easily place a hard cap on the total number of rides the player is able to build. That could force the player to put a high priority on researching new rides, and/or provide an even stricter limit if there's a limited number of rides available to research.

It could also be used to encourage players to build really big and elaborate rides of each type. For example, if you can only build two wooden coasters and that's the only coaster the scenario gives you, then you'd want to build two of the biggest, baddest wooden coasters you could imagine, in order to maximize the benefits you get from those two coasters. While the game's mechanics normally reward building lots of modest coasters, this option would instead skew things toward building just a few really awesome and well-optimized coasters.

It could really revitalize scenario play for experienced players, giving additional challenge and making the game more interesting by forcing you to build a variety of well-designed rides in order to win.

**Comment by ZeeMaji** [comment]
I think a more sensible option would be options to nerf duplicate rides rather than outright disallow them.
A scenario option that divides the value and the guest cap contribution of rides for every duplicate you have seems a bit more in line with the rest of the game than outright disallowing the construction of duplicates.

**Reply by James103** [reply]
See https://github.com/OpenRCT2/OpenRCT2/discussions/17042 for further discussion on that topic.
